docs/01_manual.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Manual
+layout: home
+---
+
+Manual
+======
+
+TABLE OF CONTENTS
+-----------------
+
+- [SYNOPSIS](#synopsis)
+- [DESCRIPTION](#description)
+- [ARGUMENTS](#arguments)
+- [INSTALLATION AND CONFIGURATION](#installation-and-configuration)
+- [BUILDING FROM SOURCE](#building-from-source)
+
+
+SYNOPSIS
+--------
+
+    easy-deposit-agreement-generator [ -s ] <datasetID> <agreement-file>
+
+
+DESCRIPTION
+-----------
+
+A command line tool that creates a pdf document containing the deposit agreement for a given dataset. The tool searches for a dataset that corresponds 
+to the given `datasetID` and uses the metadata of this dataset, as well as the personal data of the depositor to generate the deposit agreement.
+
+**Note:** the dataset needs to be in Fedora already. Newly created datasets (for example from `easy-split-multi-deposit`) need to be ingested 
+by EASY first before generating the deposit agreement.
+
+`easy-deposit-agreement-generator` uses a template with placeholders. After replacing the placeholders with actual data, the template is converted into a PDF file.
+
+Placeholder substitution is achieved using [Apache Velocity](http://velocity.apache.org/), which fills in and merges a number of template HTML 
+files that are specified in `src/main/assembly/dist/res/template/`. Besides data from the dataset, several files in `src/main/assembly/dist/res/` 
+are required, namely `dans_logo.png`, `agreement_version.txt`, `Metadataterms.properties` and `velocity-engine.properties`.
+
+Pdf generation based on the assembled HTML is done using the command line tool [WeasyPrint](http://weasyprint.org/). Note that this tool 
+requires to be installed before being used by `easy-deposit-agreement-generator`. In order to not having this installed on our computers while developing 
+on this project or projects that depend on this project, we use an SSH connection to the development server where the command gets executed. 
+During development we therefore require a different script than the one that is used in production. Follow the steps below:
+
+1. In `application.properties` set `pdf.script=localrun.sh`;
```

Comment:
these steps are VERY old!!! Even in `easy-deposit-agreement-generator` these steps no longer apply.

Instead, what happens now is that we have 2 scripts: one in `src/main/assembly/dist/res` and one in `src/test/resources/debug-config`. The former is used when for example an RPM is created, the latter is used in (potential) unit tests and in local runs using `run.sh` or `run-service.sh`.